src/ngraph/runtime/cpu/builder/matmul_bias.cpp

```diff
@@ -172,7 +176,177 @@ namespace ngraph
                 functors.emplace_back(functor);
             }
 
+            struct CblasGemmOptions
+            {
+                CblasGemmOptions(void*& da, void*& db, void*& dc)
+                    : data_a(da)
+                    , data_b(db)
+                    , data_c(dc){};
+
+                std::vector<cblas::Transpose> transa_array;
+                std::vector<cblas::Transpose> transb_array;
+                std::vector<int64_t> m_array;
+                std::vector<int64_t> n_array;
+                std::vector<int64_t> k_array;
+                std::vector<int64_t> lda_array;
+                std::vector<int64_t> ldb_array;
+                std::vector<int64_t> ldc_array;
+                std::vector<int64_t> group_sizes;
+                std::vector<float> alpha_array;
+                std::vector<float> beta_array;
+                size_t offset_a;
+                size_t offset_b;
+                size_t offset_c;
+                void*& data_a;
+                void*& data_b;
+                void*& data_c;
+                int64_t group_count;
+
+                void call(CPURuntimeContext* ctx)
+                {
+                    std::vector<float*> a_array;
+                    std::vector<float*> b_array;
+                    std::vector<float*> c_array;
```

Comment:
we know the sizes of the vectors. might be better to instantiate one with the right size instead of risking reallocs later in line 217.

Reply:
hhhmmmkaaaayyy 😄